2021-finalproject/Koonyu/check-environment/check-environment.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#check environment variable
+if [ $ROS_MASTER_URI ];then
+	echo "[Check] environment variable... Pass"
+else
+	echo "[Check] environment variable... Failed"
+fi
+
+#check address validation " http://[[1-255].[0-255].[0-255].[1-255]]:[PORT] "
+#s/REGEX/SUBSTITUTION/
+#awk is used to analyze url which is valid or not
+if (export | grep ROS_MASTER_URI | sed -E "s/.*=//" | sed -E "s/\"//" | sed -E "s/\"//" | sed -E "s/http:\/\///" | sed -E "s/:.*//" | awk -F. '{if(NF==4 && $1>0 && $1<=255 && $2>=0 && $2<=255 && $3>=0 && $3<=255 && $4>0 && $4<255) exit 0; else exit 1}');then #check if this IP contents this model " [[1-255].[0-255].[0-255].[1-255]] "
```

Comment:
What will happen if the environment variable is empty

Reply:
Before I didn't know that I could empty the environment variable with double quotation mark
Now ,  I can know that the result of emptying the environment variable is that all check failed